src/main/java/com/strandls/document/RabbitMqConnection.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * 
+ */
+package com.strandls.document;
+
+import java.io.IOException;
+
+import java.io.InputStream;
+import java.util.Properties;
+import java.util.concurrent.TimeoutException;
+
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+import com.strandls.document.util.PropertyFileUtil;
+
+/**
+ * @author Abhishek Rudra
+ *
+ */
+public class RabbitMqConnection {
+
+	private final static String QUEUE_ELASTIC = "elastic";
```

Comment:
please differentiate the queue, module wise, check if observation is using the same queue,
if yes, create a new queue, every queue has a particular job to fulfil